docs/applications/piwigo.md

```diff
@@ -0,0 +1,11 @@
+# Piwigo
+
+Homepage: [https://www.piwigo.org/](https://www.piwigo.org/)
+
+Piwigo is open source photo gallery software for the web. Designed for organisations, teams and individuals
+.
+## Usage
+
+Set `piwigo_enabled: true` in your `inventories/<your_inventory>/nas.yml` file.
+
+piwigo's web interface can be found at http://ansible_nas_host_or_ip:8033
```

Comment:
worthwhile noting that when doing the webui install the localhost will be the ip of the mysql docker container. localhost didn't work for me. Previously i used "db" with 'services' defined in my docker compose and that worked but services are not defined in the ansible tasks yml and I don't know how to do that in ansible.